test/v1_integration.py

```diff
@@ -445,6 +471,7 @@ def test_caa():
     # now be denied due to CAA.
     chisel.expect_problem("urn:acme:error:caa", lambda: chisel.issue(caa_client, caa_authzs))
 
+    challSrv.add_caa_issue({"domain": "bad-caa-reserved.com", "value": badCAA})
```

Comment:
The `challSrv.add_caa_issue` function takes two string arguments, not one dict:

```suggestion
    challSrv.add_caa_issue("bad-caa-reserved.com", badCAA)
```

I believe this is the cause of a [CI failure](https://travis-ci.org/letsencrypt/boulder/jobs/547036100):
```
Traceback (most recent call last):
  File "test/integration-test.py", line 280, in <module>
    main()
  File "test/integration-test.py", line 197, in main
    run_chisel(args.test_case_filter)
  File "test/integration-test.py", line 229, in run_chisel
    value()
  File "/go/src/github.com/letsencrypt/boulder/test/v1_integration.py", line 474, in test_caa
    challSrv.add_caa_issue({"domain": "bad-caa-reserved.com", "value": badCAA})
TypeError: add_caa_issue() takes exactly 3 arguments (2 given)
```